src/server/datasource/uniprot.js

```diff
@@ -81,7 +84,7 @@ const includeEntry = entry => {
 
 const parseXml = (filePath, onData, onEnd) => {
   let rootTag = 'entry';
-  let omitList = ['uniprot' ,'lineage', 'reference', 'evidence'];
+  let omitList = ['lineage', 'reference', 'evidence', 'copyright', 'comment'];
```

Comment:
This is a red flag.

It doesn't make sense that it would work if `<evidence>` is an omitted tag.  You have `<evidence><source><dbReference>` as the hierarchy for getting a cross-referenced ID.  It doesn't make sense that things would work with any of those tags in the omit list.

Reply:
I only take the `<dbReference >` that is the direct child of `<entry>`.

Reply:
OK, let's merge it in then